src/main/resources/META-INF/plugin.xml

```diff
@@ -77,7 +77,12 @@
       <language>PHP</language>
       <className>com.vk.kphpstorm.intentions.PrettifyPhpdocBlockIntention</className>
       <category>PHP</category>
-      <descriptionDirectoryName>PrettifyPhpdocBlockIntention</descriptionDirectoryName>
```

Comment:
Why was this line removed?

Reply:
because this line is redundant: 

<img width="949" height="224" alt="image" src="https://github.com/user-attachments/assets/057902e5-28d3-41d8-805b-7a03b091f2ed" />